packages/eslint-plugin-lit-a11y/tests/lib/rules/accessible-emoji.js

```diff
@@ -68,5 +69,5 @@ ruleTester.run('accessible-emoji', rule, {
       code: 'html`<span role="img" alt="Panda face">🐼</span>`',
       errors: [{ messageId: 'wrapEmoji' }],
     },
-  ],
+  ].map(prependLitHtmlImport),
```

Comment:
can we apply settings instead of mapping over snippets?
not sure this is possible: https://eslint.org/docs/developer-guide/nodejs-api#ruletester

Reply:
I tried to find something like that, but I couldnt find it, so mapping seemed like the most DRY solution here (even though its repeated in all the tests :p )

Reply:
it's a maintenance nightmare tho

Reply:
Just went through the docs again, and through some of the jsx plugins code as well, but I really dont see a way to simplify this further